test/format/format.sh

```diff
@@ -451,7 +460,15 @@ format()
 		echo "$name: starting job in $dir ($(date))"
 	fi
 
-	cmd="$format_binary -c "$config" -h "$dir" -1 $args quiet=1"
+	# If we're using UndoDB, append our default arguments.
+	#
+	# This script is typically left running until a failure is hit. To avoid filling up the
```

Comment:
[question] From Undo's perspective, a "failure" is a process that exits with non-0 code; other processes with 0 exit codes will not be recorded. Is this level of detail sufficient for WT's tests?

Reply:
Yeah, for `format` that's ok. We use error codes to figure out what to flag as a test failure.